Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -286,6 +319,9 @@
 
 void RemoteLayerTreeDrawingAreaProxy::asyncSetLayerContents(WebCore::PlatformLayerIdentifier layerID, ImageBufferBackendHandle&& handle, const WebCore::RenderingResourceIdentifier& identifier)
 {
+    // FIXME(site-isolation): We should use ProcessQualified for layerID to stop cross-origin
```

Comment:
I'm not sure that ProcessQualified would do this, but we do want to be careful.

Reply:
I'll drop the comments for now, but let's chat more offline about this :)